1.3/Patches/DeepTalk.xml

```diff
@@ -19,7 +22,12 @@
 			<!-- ===Dreams=== -->
 				<!-- ===Nice=== -->
 				<li>r_logentry(INITIATOR_mood>=0.6,tag=WhatDream)->Ah, [recipient]... I had such a nice dream...</li>
-
+				<li>r_logentry(INITIATOR_mood&lt;=0.6,tag=WhatDream)->I had a vivid dream last night, [recipient].</li>
+				
+			<!-- ===Random DeepTalk conversation=== -->
+				<li>r_logentry(tag=DeepTalkConvo)->Hey, [RECIPIENT_nameDef]? What do you think about [TalkTopicHeavy]?</li>
```

Comment:
Shouldn't we use [recipient] here instead? same thing for the next 2 lines

Reply:
I updated this one but forgot to save/commit it... Thank you!